src/main/java/org/broadinstitute/hellbender/engine/LocalReadShard.java

```diff
@@ -84,6 +95,13 @@ public void setDownsampler(final ReadsDownsampler downsampler) {
         this.downsampler = downsampler;
     }
 
+    /**
+     * Reads in this shard will be transformed after filtering and downsampling
+     *
+     * @param transformer read transformer to apply after read filtering and after downsampling
```

Comment:
"after read filtering and before downsampling"

Reply:
done